backend/backend/settings/base.py

```diff
@@ -164,6 +164,10 @@ def get_required_setting(
     "CELERY_BROKER_URL", f"redis://{REDIS_HOST}:{REDIS_PORT}"
 )
 
+MAX_WAIT_TIME = int(get_required_setting("MAX_WAIT_TIME", "1800"))
```

Comment:
Passing defaults for an env where we'll throw error if its not present doesn't make sense. Either remove defaults or avoid enforcing that it needs to be required

Reply:
Got it :+1: